gui/dialogs/aircraft-dialog.xml

```diff
@@ -125,11 +125,33 @@
                 <property>/engines/active-engine/oil_consumption_allowed</property>
                 <live>true</live>
                 <binding>
+                    <condition>
+                        <equals>
+                            <property>/controls/engines/active-engine</property>
+                            <value type="int">0</value>
+                        </equals>
+                    </condition>
                     <command>property-assign</command>
                     <property>/engines/active-engine/oil-level</property>
                     <value>7.0</value>
```

Comment:
Actually Dany, this whole seems unecessary to me, since as far as I understand your Nasal function is taking care of the oil level by itself (so that if the oil consumption is disables, the function will reset the oil to 7 or 8 quarts according to the engine). So maybe we should remove lines 127-148, is that correct?